libs/trainer/pyproject.toml

```
@@ -8,16 +8,25 @@ packages = [{include = "mlpe"}]
 
 [tool.poetry.dependencies]
 python = ">=3.8,<3.11"
+torch = {version = "^1.12", source = "torch"}
```

Just a quick check, as a part of this maintenance patch. Shall we remove the minor tick constraint. I noticed some of the projects/libs were installing 1.13 and others were install 1.12. It's not a problem, but I am wondering if we should just do
```suggestion
torch = {version = ">=1.12", source = "torch"}
```

---

Remind me: what is the exact difference here? I agree should definitely standardize versions